openapi.yaml

```diff
@@ -6995,6 +6995,90 @@ paths:
       - lang: CLI
         source: >
           linode-cli longview delete 789
+  /longview/plan:
+    x-linode-cli-command: longview
+    get:
+      tags:
+      - Longview
+      summary: View Longview Plan
+      description: >
+        Returns the Longview subscription plan for this user. You can send a request to the
+        [View Longview Subscription](/api/v4/longview-subscriptions-subscription-id) endpoint for details on each particular subscription plan.
+      operationId: getLongviewPlan
+      x-linode-cli-action: plan-view
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - longview:read_only
+      responses:
+        '200':
+          description: The requested Longview Subscription.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/LongviewPlan'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+              https://api.linode.com/v4/longview/plan
+      - lang: CLI
+        source: >
+          linode-cli longview plan-view
+    put:
+      tags:
+      - Longview
+      summary: Update Longview Plan
+      description: >
+        Updates the Longview subscription plan for this user. You can send a request to the
+        [View Longview Subscription](/api/v4/longview-subscriptions-subscription-id) endpoint for details on each particular subscription plan.
+      operationId: updateLongviewPlan
+      x-linode-cli-action: plan-update
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - longview:read_write
+      requestBody:
+        description: Update your Longview subscription plan.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/LongviewPlan'
+      responses:
+        '200':
+          description: The requested Longview Subscription.
+          content:
+            application/json:
+              schema:
+                properties:
+                  longview_subscription:
+                    type: string
+                    description: >
+                      The Longview subscription plan you are currently subscribed to.
+
+
+
+                      To update your subscription plan send a request to [Update Longview Plan](/api/v4/longview-plan/#put).
+                      The value must be a [Longview Subscription ID](/api/v4/longview-subscriptions) or `null`.
```

Comment:
Can we specifically call out that if a null is passed, it will cancel the longview subscription?